test/data/browsercharm.json

```diff
@@ -1,352 +1,455 @@
 {
-  "charm": {
```

Comment:
This is changing the test json data into the apiv4 processed data format - you can skip this file.